### ISSUE TITLE
Doesn't play well with jQuery 3.1.0

### ISSUE THREAD

**Issue by asantos** [tuupola/jquery_chained#47]
If used with Select2 and jQuery 3.1.0, it throws this error: 

`jquery.min.js:2 Uncaught TypeError: e(...).size is not a function`

More details here:

```
jQuery.Deferred exception: e(...).size is not a function TypeError: e(...).size is not a function
    at i (http://localhost/js/min/common.min.1469728052.js:3:12578)
    at HTMLSelectElement.<anonymous> (http://localhost/js/min/common.min.1469728052.js:3:12851)
    at Function.each (http://localhost/scripts/jquery-3.1.0/jquery.min.js:2:2815)
    at r.each (http://localhost/scripts/jquery-3.1.0/jquery.min.js:2:1003)
    at HTMLSelectElement.<anonymous> (http://localhost/js/min/common.min.1469728052.js:3:12709)
    at Function.each (http://localhost/scripts/jquery-3.1.0/jquery.min.js:2:2815)
    at r.each (http://localhost/scripts/jquery-3.1.0/jquery.min.js:2:1003)
    at r.e.fn.chained (http://localhost/js/min/common.min.1469728052.js:3:12084)
    at HTMLDocument.<anonymous> (http://localhost/js/min/index.min.1469728055.js:1:443)
    at j (http://localhost/scripts/jquery-3.1.0/jquery.min.js:2:29568) undefined
jquery.min.js:2 Uncaught TypeError: e(...).size is not a functioni @ common.min.1469728052.js:3(anonymous function) @ common.min.1469728052.js:3each @ jquery.min.js:2each @ jquery.min.js:2(anonymous function) @ common.min.1469728052.js:3each @ jquery.min.js:2each @ jquery.min.js:2e.fn.chained @ common.min.1469728052.js:3(anonymous function) @ index.min.1469728055.js:1j @ jquery.min.js:2k @ jquery.min.js:2
```

**Comment by asantos** [comment]
I fixed it locally by replacing:

`if (1 === $("option", child).size() && $(child).val() === "") {`

With:

`if (1 === $("option", child).length && $(child).val() === "") {`

**Comment by ChristophBoget** [comment]
What's the status of this bug?  Are there plans to get this plug-in to play nice with jQuery 3.x?

**Comment by tuupola** [comment]
Totally forgot this. Will look into it.

**Comment by tuupola** [comment]
Fixed in #52.
